CONTRIBUTING.md

```diff
@@ -316,7 +298,118 @@ You entry will look something like the following when edited:
 
 The `title` property is displayed above the editor, and should be of the form: "JavaScript Demo: {item}", where {item} is the name of the item that the example is for. If you're not sure what to use for {item}, use the title of the page.
 
-### Testing
+Once you've finished writing the example, see the [Testing](#testing) section for the next step.
```

Comment:
Yes, but it's coming through a different path ("Contributing a CSS example"). There's also one for the "Contributing a JavaScript example" path. In the previous version the whole "testing" section was duplicated between CSS and JS: in this edit I chose to factor it out, but then we need pointers to that section from each of the CSS, JS, and HTML sections.

The structure of this document is now a little complicated, and it might be time to split it up, but I've chosen not to do this yet.

Reply:
Ah, I thought it might be the diff